grpc/src/main/java/com/linecorp/armeria/server/grpc/DeferredListener.java

```diff
@@ -74,105 +73,91 @@
             }
 
             this.delegate = delegate;
-            try {
-                for (Consumer<Listener<I>> task : pendingTasks) {
-                    task.accept(delegate);
+            while (inLoop = true) {
```

Comment:
This is a little bit confusing and even looks like a bug although it's not.
```suggestion
            for (;;) {
                inLoop = true;
```